espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -199,6 +201,15 @@ def add_arguments(parser):
         group.add_argument(
             "--dunits", default=320, type=int, help="Number of decoder hidden units"
         )
+        # Non-autoregressive training
+        group.add_argument(
+            "--dmode",
```

Comment:
what does "dmode" mean?

Reply:
It means "decoder training mode", specifying whether decoder is trained in the autoregressive or non-autoregressive manner.